features/polyExplorer/src/screens/entityDetails/entityDetails.jsx

```diff
@@ -219,6 +222,14 @@ const EntityDetails = () => {
                                     <CompanyRevenueChart
                                         annualRevenues={entity.annualRevenues}
                                     />
+                                    <SourceInfoButton
+                                        source={i18n.t(
+                                            "common:source.polyPedia"
+                                        )}
+                                        //  TO DO: add correct path (maybe HTRT-Screen doesn't exist yet?)
```

Comment:
As a principle, in think that these TODOs should be avoided, and substituted, where possible, with Jira tickets that give more context (and can eventually be fixed by anyone)
(This is a pet peeve of mine, so disregard if possible)